corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -413,6 +396,11 @@
           callback: function (ko, mapping) {
             ko.mapping = mapping;
           },
+          shim: {
```

Comment:
Didn't notice this earlier - can you move this into `hqwebapp/js/requirejs_config.js`? The only requirejs config in `base.html` is what has to be here.